src/dialects/abstract/query-generator.js

```diff
@@ -2861,6 +2861,22 @@ Only named replacements (:name) are allowed in literal() because we cannot guara
 
         return this._joinKeyValue(key, this.escape(pattern, undefined, options), comparator, options.prefix);
       }
+
+      case Op.anyKeyExists: {
```

Comment:
These two cases are the same right? We should combine these then to not have duplicated code

Reply:
Thanks